cardano-node/cardano-node.cabal

```diff
@@ -7,7 +7,6 @@ license:        Apache-2.0
 license-files:
   LICENSE
   NOTICE
-copyright:      2019 Input Output (Hong Kong) Ltd.
```

Comment:
In some of our other repos, we appear to keep this line. Is it actually an issue for it to remain?

Reply:
Apparently: 

input-output-hk/plutus#635

> Remove Cabal copyright field.
> There is a copyright notice in NOTICE which this would conflict
> with.